windows-driver-docs-pr/install/inf-addservice-directive.md

```diff
@@ -244,6 +245,14 @@ For information about security descriptor strings, see [Security Descriptor Defi
 
 For more information about how to specify security descriptors, see [Creating Secure Device Installations](creating-secure-device-installations.md).
 
+**RequiredPrivileges**=*privilege-name*[**,**_privilege-name_]...
+> [!NOTE]
+> This value can only be used for *Win32 Services* and is only available on Windows 7 and above.
+
+Each *privilege-name* in the list is the name of the privilege.  See [Privilege Constants (Windows)](/windows/win32/secauthz/privilege-constants) for a list of the privilegs names. For each privilege name, only the text name is required, *For Example* The privilege name should be written as "SetAuditPrivilege" but not SE_AUDIT_NAME.
+
+For more information on service required privileges, see [SERVICE_REQUIRED_PRIVILEGES_INFO (Windows)](/windows/win32/api/winsvc/ns-winsvc-service_required_privileges_infow)
```

Comment:
Should have a period at the end of this sentence.

Reply:
Sure. This has been modified accordingly